test/jdk/javax/swing/JTree/6263446/bug6263446.java

```diff
@@ -166,9 +181,11 @@ private static void createAndShowGUI() {
 
 
         frame.getContentPane().add(tree);
-        frame.pack();
+        frame.setAlwaysOnTop(true);
         frame.setLocationRelativeTo(null);
+        frame.pack();
```

Comment:
pack() after the setLocationRelativeTo() may move the frame out of the center of the screen, is it intentional?

Reply:
it doesn't and I have checked

Reply:
Please recheck, setLocationRelativeTo() calculates the location based on the size of the current frame, which is wrong before the pack. I just run it and the frame is sifted to the right part of the screen.

Reply:
WHich platform it is shifted?

Reply:
I guess on any, I have tested on windows, and now on macOS. Add these lines to the end of method, and check how the frame will jump to the center:
```
        frame.setLocationRelativeTo(null);
        frame.pack();
        frame.setVisible(true);
      + robot.delay(3000);
      + frame.dispose();
      + frame.pack();
      + frame.setLocationRelativeTo(null);
      + frame.setVisible(true);
```

Reply:
OK. Test updated
 (although I dont think it would have mattered in this specific case for such a small JTree widget as can be seen from my testing CI jobs in all platforms)..